### PR TITLE
Editor: Add TaskTagger so TODOs get highlighted

### PR DESCRIPTION
**Pull request by sandyarmstrong**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/918990

### CODE REVIEW

File: main/src/addins/MonoDevelop.TextEditor/MonoDevelop.TextEditor/ThemeToClassification.cs [hunk 1]
```diff
@@ -223,6 +224,7 @@ void UpdateEditorFormatMap (string appearanceCategory)
 			CreateResourceDictionary (editorFormat, defaultSettings, "RulerFormat", EditorThemeColors.Ruler);
 			CreateResourceDictionary (editorFormat, defaultSettings, "NavigableSymbolFormat", EditorThemeColors.Link, EditorFormatDefinition.ForegroundColorId);
 			CreateResourceDictionary (editorFormat, defaultSettings, "urlformat", EditorThemeColors.Link, EditorFormatDefinition.ForegroundColorId);
+			CreateResourceDictionary (editorFormat, defaultSettings, "urlformat", EditorThemeColors.Link, EditorFormatDefinition.ForegroundColorId);
```
Comment (sandyarmstrong) [inline main/src/addins/MonoDevelop.TextEditor/MonoDevelop.TextEditor/ThemeToClassification.cs]: Whoops, need to delete this testing code.

File: main/src/addins/MonoDevelop.TextEditor/MonoDevelop.TextEditor.Cocoa/MonoDevelop.TextEditor.Cocoa.csproj [hunk 2]
```diff
@@ -63,6 +63,7 @@
     <ProjectReference Include="$(VSEditorCoreDirectory)src\Editor\Text\Impl\VisibleWhitespace\VisibleWhitespaceImpl.csproj" Private="True" />
     <ProjectReference Include="$(VSEditorCoreDirectory)src\Editor\Text\Impl\GlyphMargin\GlyphMarginImpl.csproj" Private="True" />
     <ProjectReference Include="$(VSEditorCoreDirectory)src\Editor\Text\Impl\Structure\StructureImpl.csproj" Private="true" />
+    <ProjectReference Include="$(VSEditorCoreDirectory)src\Editor\Text\Impl\TaskTagger\TaskTagger.csproj" Private="true" />
```
Comment (avodovnik) [inline main/src/addins/MonoDevelop.TextEditor/MonoDevelop.TextEditor.Cocoa/MonoDevelop.TextEditor.Cocoa.csproj]: Just out of curiosity, I thought this is coming from the new consolidated project. Guess not?

Comment (sandyarmstrong) [reply main/src/addins/MonoDevelop.TextEditor/MonoDevelop.TextEditor.Cocoa/MonoDevelop.TextEditor.Cocoa.csproj]: We're not actually using the consolidated assembly yet.

File: main/src/addins/MonoDevelop.TextEditor/MonoDevelop.TextEditor/ThemeToClassification.cs (hunk 1 above)
Comment (sandyarmstrong) [reply main/src/addins/MonoDevelop.TextEditor/MonoDevelop.TextEditor/ThemeToClassification.cs]: I fixed.

**Pull request merged**